lib/ReactViews/Credits/MapCredits/MapCredits.props.ts

```diff
@@ -0,0 +1,8 @@
+import { ICredit } from "../Credit.type";
+import GlobeOrMap from "../../../Models/GlobeOrMap";
+
+export interface IMapCreditsProps {
```

Comment:
perfectly fine for me, just got used to this on another project it is really easy to find props. In Terria most of the react files have props, updated styled components with custom styling needed for it, and then file starts to grow. But it would be probably too much change to go with this

I don't see this either `or naming it {componentName}Types.ts`